src/jens/test/test_git_wrapper.py

```diff
@@ -154,3 +154,24 @@ def test_get_refs_non_existing_repository(self):
     def test_get_refs_not_repository(self):
         not_repo_path = create_folder_not_repository(self.sandbox_path)
         self.assertRaises(JensGitError, git_wrapper.get_refs, not_repo_path)
+
+    def test_get_head_existing_repository(self):
```

Comment:
Couldn't run individual tests with e.g.

```
nosetests jens.test.test_git_wrapper.GitWrapperTest.test_get_head_existing_repository
```

Is it expected? Am I doing something wrong?